db/migrate/20160720100123_create_recipes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRecipes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :recipes do |t|
+      t.string :name
+      t.text :description
```

Comment:
you should add valdation not only in model but inside db too, check previous @jeathli commit (PR)